packages/atomic/cypress/integration/facets/category-facet/category-facet-assertions.ts

```diff
@@ -54,9 +53,7 @@ export function assertNoPathInUrl() {
 
 export function assertLogFacetSelect(path: string[]) {
   it('should log the facet selection to UA', () => {
-    cy.wait(RouteAlias.analytics).then((intercept) => {
-      const analyticsBody = intercept.request.body;
-      expect(analyticsBody).to.have.property('actionCause', 'facetSelect');
+    cy.expectSearchEvent('facetSelect').then((analyticsBody) => {
```

Comment:
I added `cy.expectSearchEvent` and `cy.expectClickEvent`, which is defined below in `commands.ts`.